feather/worldgen/src/lib.rs

```diff
@@ -141,8 +141,8 @@ impl WorldGenerator for ComposableGenerator {
 
         let mut chunk = Chunk::new(position);
 
-        for x in 0..16 {
-            for z in 0..16 {
+        for x in 0..4 {
+            for z in 0..4 {
```

Comment:
This... doesn't seem right but I don't claim to 100% understand what's going on. Why is this change necessary?